modules/starfish/2.0/config/default.yaml

```diff
@@ -0,0 +1,51 @@
+lcr-modules:
+    
+    starfish:
+
+        dirs: 
+            
+            _parent: "__UPDATE__"
```

Comment:
Is this documented? Do you want to provide a best practices example? I might not be asking this if this was adopted in the demo.

Reply:
Ah yes I will add a comment here, thanks.